### PR TITLE
Collateral amounts in confirm page

### PR DESCRIPTION
**Pull request by aminlatifi**
Updated collateram amount and value in confirm page

### CODE REVIEW

**Review by lovelgeorge99 — APPROVED** [review]

File: src/helpers/currency.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const res = await fetch(
+      `https://api.coingecko.com/api/v3/simple/price?ids=${coingeckoId}&vs_currencies=usd`
+    );
+    const data = await res.json();
+    return parseFloat(data[coingeckoId].usd) || 1;
+  } catch (error) {
+    console.error("Error fetching token price:", error);
+    return 1;
+  }
+};
```
Comment (MohammadPCh) [inline src/helpers/currency.ts:27]: It's better to move this to services folder.

File: src/components/TokenForm/ConfirmStep.tsx [hunk 2]
```diff
@@ -24,6 +26,17 @@ const ConfirmStep: React.FC<{ onNext: () => void; onBack: () => void }> = ({
...
+  const collateralAmount = +config.bondingCurveParams.initialCollateralSupply;
+  const collateralUsdValueString: string = useMemo(() => {
+    return polTokenPrice.isSuccess
+      ? formatCurrencyAmount(collateralAmount * polTokenPrice.data)
+      : "-";
+  }, [collateralAmount, polTokenPrice.isSuccess, polTokenPrice.data]);
+
+  useEffect(() => {
+    console.log("POL Token Price: ", polTokenPrice);
+  }, [polTokenPrice]);
```
Comment (MohammadPCh) [inline src/components/TokenForm/ConfirmStep.tsx:39]: It's better to remove this, wdyt?

File: src/components/TokenForm/ConfirmStep.tsx [hunk 3]
```diff
@@ -24,6 +26,17 @@ const ConfirmStep: React.FC<{ onNext: () => void; onBack: () => void }> = ({
   const { handleSubmit, formState } = methods;
   const { deploy, prep, requestedModules, inverter } = useDeploy();
   const collateralCheck = useCollateralCheck();
+  const polTokenPrice = usePolTokenPrice();
+  const collateralAmount = +config.bondingCurveParams.initialCollateralSupply;
+  const collateralUsdValueString: string = useMemo(() => {
+    return polTokenPrice.isSuccess
+      ? formatCurrencyAmount(collateralAmount * polTokenPrice.data)
+      : "-";
+  }, [collateralAmount, polTokenPrice.isSuccess, polTokenPrice.data]);
```
Comment (MohammadPCh) [inline src/components/TokenForm/ConfirmStep.tsx:35]: it's not necessary to use useMemo for this.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
Thanks @aminlatifi Just small suggestions.

**Pull request merged**